apps/examples/wifi_manager/wm_test/wm_test.c

```diff
@@ -938,6 +938,38 @@ void _wt_process(int argc, char *argv[])
 	WT_TEST_FUNC_SIGNAL;
 }
 
+#if defined(CONFIG_ENABLE_HOMELYNK) && (CONFIG_ENABLE_HOMELYNK == 1)
+int _wt_parse_bridge(struct wt_options *opt, int argc, char *argv[])
+{
+	if (argc < 3) {
```

Comment:
```suggestion
	if (argc < 4) {
```
if you using argv[3], this condition checking should be < 4